guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -3,31 +3,51 @@
 
-Configuring the {SmartProxyServer} has two basic requirements:
+Configuring {SmartProxyServer} has two basic requirements:
 
 . Configuring network services. This includes:
   * Content delivery services
   * Network services (DHCP, DNS, and TFTP)
   * Puppet configuration
 . Defining network resource data in {ProjectServer} to help configure network interfaces on new hosts.
 
-The following instructions have similar applications to configuring standalone {SmartProxyServer}s managing a specific network. To configure {Project} to use external DHCP, DNS, and TFTP services, see link:{BaseURL}installing_satellite_server_from_a_connected_network/configuring_external_services[Configuring External Services] in _Installing {ProjectServer} from a Connected Network_.
+The following instructions have similar applications to configuring standalone {SmartProxies} managing a specific network. To configure {Project} to use external DHCP, DNS, and TFTP services, see link:{BaseURL}installing_satellite_server_from_a_connected_network/configuring_external_services[Configuring External Services] in _Installing {ProjectServer} from a Connected Network_.
 
-.DHCP Ranges
-You can define the same DHCP range in {ProjectServer} for both Discovered and Provisioned systems, but use a separate range for each service within the same subnet.
+[[networking_resources]]
+.Network Resources
+
+{Project} contains networking resources that you must set up and configure to create a host. {Project} includes the following networking resources:
+
+Domain::
+You must assign every host that is managed by {Project} to a domain. Using the domain, {Project} can manage A, AAAA and PTR records. Even if you do not want {Project} to manage your DNS servers, you still must create and associate at least one domain. Domains are included in the naming conventions {Project} hosts, for example, a host `test123` in the `example.com` domain has the host name `test123.example.com`.
+
+Subnet::
+You must assign every host managed by {Project} to a subnet. Using subnets, {Project} can then manage IPv4 reservations. If there are no reservation integrations, you still must create and associate at least one subnet. When you manage a subnet in {Project}, you cannot create DHCP records for that subnet outside of {Project}. In {Project} can manage IP addresses with one of the following options:
+
+In {Project}, you can set IP Address Management (IPAM) for a subnet.
+
+* *DHCP*: the DHCP {SmartProxy} manages the assignment of IP addresses by finding the next available IP address starting from the first. Before assigning an IP address, {SmartProxy} pings ICMP and TCP to verify whether the IP address is in use.
```

Comment:
s/pings ICMP and TCP to verify whether the IP address is in use./sends an ICMP and TCP ping to verify whether the IP address is in use.